pipelines/DRPFakes.yaml

```diff
@@ -8,7 +8,7 @@ imports:
     - faro_step5
 
 tasks:
-  processCcdWithFakes:
+  detectorVisitFakes:
```

Comment:
This new name doesn't seem to capture what this task does. Would `singleFrameWithFakes` work better here?